app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -52,10 +53,12 @@ class _ConvoCardState extends ConsumerState<ConvoCard> {
           ref
               .read(currentConvoProvider.notifier)
               .update((state) => widget.room);
-          context.pushNamed(
-            Routes.chatroom.name,
-            pathParameters: {'roomId': roomId},
-          );
+          if (!isDesktop(context)) {
```

Comment:
not fond of this, but good enough for now. let's work on the actual routing in another PR